src/connectors/panoptic/panoptic.ts

```diff
@@ -884,12 +889,23 @@ export class Panoptic {
     try {
       const panopticpool = this.PanopticPool;
       const panopticPoolContract = new Contract(panopticpool, panopticPoolAbi.abi, wallet);
+      const gasEstimate: number = (await panopticPoolContract.estimateGas["burnOptions(uint256,uint256[],int24,int24)"](
+        burnTokenId,
+        newPositionIdList,
+        tickLimitLow,
+        tickLimitHigh
+      )).toNumber();
+      const gasLimit: number = Math.ceil(this.gasFactor * gasEstimate);
+      if (gasLimit > this.gasLimitEstimate) {
+        return new Error(`Error on executeBurn: Gas limit exceeded, gas estimate limit (${gasLimit}) greater than tx cap (${this.gasLimitEstimate})...`);
+      }
       const tx: ContractTransaction = await panopticPoolContract["burnOptions(uint256,uint256[],int24,int24)"](
         burnTokenId,
         newPositionIdList,
         tickLimitLow,
         tickLimitHigh,
         { gasLimit: this.gasLimitEstimate }
+        // { gasLimit: BigNumber.from(gasLimit) }
```

Comment:
Still need to uncomment and swap this one in :)

Reply:
Done: 815a41290b4e626dacf19b121a5bbd394dda8915